packages/hint/src/lib/types.ts

```diff
@@ -9,7 +9,6 @@ export * from './types/events';
 export * from './types/formatters';
 export * from '@hint/utils/dist/src/types/json-parser';
```

Comment:
I believe we wanted to remove the "re-exports" from hint, right? I think there are a couple more in this file that need to be taken care of.

Reply:
In this PR, I'm only removing the references if we are extracting them to a new package, if not, it will be done as part of https://github.com/webhintio/hint/issues/3166

Reply:
SGTM, we should do this sooner rather than later 😅